packages/govern-console/src/containers/DAO/DaoHomePage.tsx

```diff
@@ -18,7 +19,7 @@ const DaoHomePage: React.FC = () => {
         <Redirect exact from={path} to={`${path}actions`} />
         <ApmRoute path={`${path}actions`} render={() => <div>Actions</div>} />
         <ApmRoute path={`${path}finance`} render={() => <div>Finance</div>} />
-        <ApmRoute path={`${path}settings`} render={() => <div>Settings</div>} />
+        <ApmRoute path={`${path}settings`} render={() => <DaoSettings />} />
```

Comment:
The final routing setup for these pages (Actions, Finance, Settings) is being done on the feature/dao-side-card branch. 
1. Yes, it's going to be removed. 
2. The reason we're using the render props pattern here is because we will be passing down the list of actions (proposals, transactions) to the Actions component. All other components that don't need props from the `DaoHomePage` will be using the `component` prop as apposed to `render`.